psi4/driver/inputparser.py

```diff
@@ -796,7 +789,9 @@ def future_replace(m):
 
     # Move up the psi4.core namespace
     for func in dir(core):
-        temp = temp.replace("psi4." + func, "psi4.core." + func)
+        # Except fns that have valid wrappers in driver
+        if func not in ['set_memory']:
+            temp = temp.replace("psi4." + func, "psi4.core." + func)
```

Comment:
Yes, it is awkward, and renaming is a better idea than the excepting I did. How about `psi4.core.set_memory_bytes(bytes)` and `psi4.core.get_memory()`. The symmetry of the core fn names is lost, but for the most part, the user still sees `psi4.set_memory()` and `psi4.get_memory()`.